ui/imports/shared/views/NetworkCardsComponent.qml

```diff
@@ -22,24 +22,22 @@ Item {
     property double amountToSend: 0
     property double requiredGasInEth: 0
     property bool errorMode: {
-        if(customMode) {
-            return (d.customAmountToSend > amountToSend) || (d.customAmountToSend < amountToSend) ||
-                    (d.customAmountToReceive > amountToSend) || (d.customAmountToReceive < amountToSend)
-        }
-        else {
-            return  !d.thereIsApossibleRoute
-        }
+        if(amountToSend === 0)
```

Comment:
What if `customAmountToSend` is bigger when `amountToSend` is `0`? Shouldn't this be an error case? Or this won't be ever the case?

Reply:
I removed this condition it is incorrect